scripts/test-npm-package.js

```
@@ -0,0 +1,160 @@
+        });
+        return result.trim();
+    } catch (err) {
+        throw new Error(`Command failed: ${command}\n${err.message}`);
+    }
+}
+
+async function testPackage() {
+    log('Testing Serpen npm package...');
+
+    // Create a temporary directory for testing
+    const testDir = path.join(os.tmpdir(), 'serpen-npm-test-' + Date.now());
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)